aspnetcore/tutorials/first-mvc-app/start-mvc.md

```diff
@@ -16,7 +15,7 @@ uid: tutorials/first-mvc-app/start-mvc
 
 By [Rick Anderson](https://twitter.com/RickAndMSFT)
 
-This tutorial will teach you the basics of building an ASP.NET Core MVC web app using [Visual Studio 2017](https://www.visualstudio.com/).
+This tutorial will teach you the basics of building an ASP.NET Core MVC web app using [Visual Studio 2017](https://www.visualstudio.com/). [Razor Pages](xref:mvc/razor-pages/index) is the recommended way to build UI for web applications.
```

Comment:
It's important to clarify that Razor Pages is the recommended way for ASP.NET Core 2.0, since it didn't exist before that point.